varats/varats/experiments/vara/feature_region_verifier_experiment.py

```diff
@@ -0,0 +1,164 @@
+from varats.data.reports.region_verification_report import (
+    RegionVerificationReport as FRR,
+)
+from varats.experiment.experiment_util import (
+    ExperimentHandle,
+    exec_func_with_pe_error_handler,
+    VersionExperiment,
+    get_default_compile_error_wrapped,
+    get_varats_result_folder,
+    wrap_unlimit_stack_size,
+    create_default_compiler_error_handler,
+    create_default_analysis_failure_handler,
+)
+from varats.experiment.wllvm import (
+    RunWLLVM,
+    get_bc_cache_actions,
+    get_cached_bc_file_path,
+    BCFileExtensions,
+)
+from varats.report.report import BaseReport
+from varats.report.report import FileStatusExtension as FSE
+from varats.report.report import ReportSpecification
+from varats.utils.settings import bb_cfg
+
+
+class FeatureRegionGeneration(actions.Step):  # type: ignore
+    """Analyse a project with VaRA and compare dominator and if-region created
+    FeatureRegions."""
+
+    NAME = "FeatureRegionGeneration"
+    DESCRIPTION = "Analyse the bitcode with -vara-PFTD and -vara-PFTDD -vara-FR-verifier"
+
+    def __init__(self, project: Project, experiment_handle: ExperimentHandle):
+        super().__init__(obj=project, action_fn=self.analyze)
+        self.__experiment_handle = experiment_handle
+
+    def analyze(self) -> actions.StepResult:
+        """
+        This step performs the actual analysis with the correct command line
+        flags.
+
+        Flags used:
+        """
+
+        if not self.obj:
+            return actions.StepResult.ERROR
+        project = self.obj
+
+        # Add to the user-defined path for saving the results of the
+        # analysis also the name and the unique id of the project of every
+        # run.
+        vara_result_folder = get_varats_result_folder(project)
+
+        for binary in project.binaries:
+            result_file = self.__experiment_handle.get_file_name(
+                FRR.shorthand(),
+                project_name=str(project.name),
+                binary_name=binary.name,
+                project_revision=project.version_of_primary,
+                project_uuid=str(project.run_uuid),
+                extension_type=FSE.SUCCESS,
+            )
+
+            opt_params = [
+                "-vara-PTFD", "-vara-PTFDD", "-vara-FR-verifier", "-o", "/dev/null",
+                get_cached_bc_file_path(
+                    project, binary, [
+                        BCFileExtensions.NO_OPT, BCFileExtensions.TBAA,
+                        BCFileExtensions.FEATURE
+                    ]
+                )
+            ]
+
+            run_cmd = opt[opt_params]
+
+            run_cmd = wrap_unlimit_stack_size(run_cmd)
+
+            run_cmd = run_cmd > f"{vara_result_folder}/{result_file}"
+
+            exec_func_with_pe_error_handler(
+                run_cmd,
+                create_default_analysis_failure_handler(
+                    self.__experiment_handle, project, FRR, Path(vara_result_folder)
```

Comment:
```suggestion
                    self.__experiment_handle, project, FRR, vara_result_folder
```
The result folder is already a path